addons/medical_treatment/functions/fnc_placeInBodyBag.sqf

```diff
@@ -31,7 +31,7 @@ if (!local _patient) exitWith {
 
 if (alive _patient) then {
     TRACE_1("Manually killing with setDead",_patient);
-    [_patient, "buried_alive"] call EFUNC(medical_status,setDead);
+    [_patient, "buried_alive", _medic] call EFUNC(medical_status,setDead);
```

Comment:
Any possible unwanted effects of having medics considered the killer of teammates?

Does this new command variation influence rating (could cause AI to consider them sideEnemy)? @dedmen

Reply:
It does affects rating, per Dedmen's comment in A3 discord, #community_wiki.